Description: Add install rules for man page even when not building DOC
Author: Mathieu Malaterre <malat@debian.org>
Origin: upstream
Forwarded: no
Last-Update: 2022-01-14

Index: openjpeg2/src/bin/jp2/CMakeLists.txt
===================================================================
--- openjpeg2.orig/src/bin/jp2/CMakeLists.txt
+++ openjpeg2/src/bin/jp2/CMakeLists.txt
@@ -77,7 +77,6 @@ foreach(exe opj_decompress opj_compress
   endif()
 endforeach()
 
-if(BUILD_DOC)
 # Install man pages
 install(
   FILES       ${OPENJPEG_SOURCE_DIR}/doc/man/man1/opj_compress.1
@@ -85,4 +84,3 @@ install(
               ${OPENJPEG_SOURCE_DIR}/doc/man/man1/opj_dump.1
   DESTINATION ${OPENJPEG_INSTALL_MAN_DIR}/man1)
 #
-endif()
Index: openjpeg2/CMakeLists.txt
===================================================================
--- openjpeg2.orig/CMakeLists.txt
+++ openjpeg2/CMakeLists.txt
@@ -131,10 +131,10 @@ if(NOT OPENJPEG_INSTALL_INCLUDE_DIR)
   set(OPENJPEG_INSTALL_INCLUDE_DIR "include/${OPENJPEG_INSTALL_SUBDIR}")
 endif()
 
-if(BUILD_DOC)
 if(NOT OPENJPEG_INSTALL_MAN_DIR)
   set(OPENJPEG_INSTALL_MAN_DIR "share/man/")
 endif()
+if(BUILD_DOC)
 
 if(NOT OPENJPEG_INSTALL_DOC_DIR)
   set(OPENJPEG_INSTALL_DOC_DIR "share/doc/${OPENJPEG_INSTALL_SUBDIR}")
Index: openjpeg2/src/lib/openjp2/CMakeLists.txt
===================================================================
--- openjpeg2.orig/src/lib/openjp2/CMakeLists.txt
+++ openjpeg2/src/lib/openjp2/CMakeLists.txt
@@ -127,12 +127,10 @@ install(FILES openjpeg.h opj_stdint.h
   DESTINATION ${OPENJPEG_INSTALL_INCLUDE_DIR} COMPONENT Headers
 )
 
-if(BUILD_DOC)
 # install man page of the library
 install(
   FILES       ${OPENJPEG_SOURCE_DIR}/doc/man/man3/libopenjp2.3
   DESTINATION ${OPENJPEG_INSTALL_MAN_DIR}/man3)
-endif()
 
 if(BUILD_LUTS_GENERATOR)
 # internal utility to generate t1_luts.h and t1_ht_luts.h (part of the jp2 lib)
